docs/docs/Release_Notes/4.0.0/4.0.0_Breaking_Changes.md

```diff
@@ -0,0 +1,36 @@
+# Breaking Changes - v.4.0.0
+
+## New Language Requirement
+[Snapshot Parameters Definition](/References/Schemas_and_Autogenerated_Classes.md#snapshotparametersdefinition-schema) allows plugin authors to define [snapshot parameters](/References/Glossary.md#snapshot-parameters) which can be displayed to an end-user whenever a linked source snapshot is taken.
+
+### What is affected
+All plugins built with v2.0.0 or below will be affected. The [Schema](/References/Schemas) must contain a `snapshotParametersDefinition`.
```

Comment:
wasn't this already in 3.1.0?

Reply:
oh jeez this is just some left over copy paste, forgot to edit this file. just updated the PR